packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -226,7 +226,7 @@ export default defineComponent({
 
     const computedInputAttributes = computed(() => ({
       ...computedChildAttributes.value,
-      ...pick(props, ['type', 'disabled', 'readonly', 'placeholder', 'pattern', 'inputmode', 'minlength', 'maxlength']),
+      ...pick(props, ['type', 'disabled', 'readonly', 'placeholder', 'pattern', 'inputmode', 'minlength', 'maxLength']),
```

Comment:
Same for minLength I think.
```suggestion
      ...pick(props, ['type', 'disabled', 'readonly', 'placeholder', 'pattern', 'inputmode', 'minLength', 'maxLength']),
```

Also, would be nice to handle maxlength and minlength for attrs (similar to native input, but it is nice to have, not required)